zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -56,6 +59,17 @@
 }
 
 
+class OppleCluster(XiaomiAqaraE1Cluster):
+    """Opple cluster."""
+
+    attributes = {
+        0x0009: ("mode", types.uint8_t, True),
+    }
+    attr_config = {0x0009: 0x01}
```

Comment:
Do you want to set the `mode` attribute (`0x09`) to `0x01 / `True`when pairing/binding?
If so, it would need to be specified in `bind()` and have `SKIP_CONFIGURATION` removed.
I'm assuming it doesn't matter though. If so, remove the `attr_config` line.

Reply:
tested all didnt make a difference so i removed the lines still works